src/hotspot/share/gc/shared/satbMarkQueue.cpp

```diff
@@ -321,7 +321,7 @@ void SATBMarkQueueSet::print_all(const char* msg) {
       _qset(qset), _buffer(buffer) {}
 
     virtual void do_thread(Thread* t) {
-      os::snprintf(_buffer, SATB_PRINTER_BUFFER_SIZE, "Thread: %s", t->name());
+      os::snprintf_checked(_buffer, SATB_PRINTER_BUFFER_SIZE, "Thread: %s", t->name());
```

Comment:
```suggestion
      (void) os::snprintf(_buffer, SATB_PRINTER_BUFFER_SIZE, "Thread: %s", t->name());
```
Can this be a JavaThread with an arbitrarily long name()?

Reply:
Yes it could - thanks. It is likely enough that we could hit this during testing that asserting if it happens is not really useful. Long Java thread names are not necessarily errors to be fixed.

Reply:
In addition, if we were to hit such a name we are not likely to increase the buffer size just for this extreme case.